subcommittees/proposal_instructions.md

```diff
@@ -0,0 +1,35 @@
+##Software Carpentry Subcommittee & Task Force Program
+
+One of the greatest things about Software Carpentry is the huge diversity of people and ideas that have come together to make this project what it is: a worldwide collaboration of folks working together to build software skills. Your contributions to the community have made Software Carpentry a huge success - but we can go even further. To meet the needs of the growing global community, the 2016 Steering Committee is creating the opportunity for you to make Software Carpentry what your community needs it to be, via our **newly revamped Subcommittee & Task Force Program**.
```

Comment:
On Thu, Apr 21, 2016 at 06:21:08PM -0700, Bill Mills:
“The logic for this structure is to provide a simple document up front
[snip reasonable stuff].”

I think it's just a matter of degree.  A line or two of similar
content around a link to the other document is helpful context.
Paragraphs of similar content are confusing noise ;).